### PR TITLE
Implement the Installation flow

### PR DESCRIPTION
**Pull request by ogwurujohnson**
## What does this PR do?
 - persist organization information to the Database
 - complete installation flow

### CODE REVIEW

File: data/migrations/20190820170537_organizations.js [hunk 1]
```diff
@@ -1,17 +1,21 @@
 exports.up = function(knex) {
```
Comment (erozonachi) [inline data/migrations/20190820170537_organizations.js]: Changes to a DB table's schema should be done in a new migration file

File: data/migrations/20190828030942_alter-organization-table.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  return knex.schema.table('organizations', tbl => {
+    tbl.text('channel_name').notNullable();
+    tbl.text('channel_id').notNullable();
+    tbl.text('access_token').notNullable();
+    tbl.timestamp('installation_date').defaultTo(knex.fn.now());
+  });
+};
+
+exports.down = function(knex) {
+  return knex.schema.dropTableIfExists('organizations');
```
Comment (samarv) [inline data/migrations/20190828030942_alter-organization-table.js]: Migrations are to make sure everyone has same db schema
migrate down should UNDO the changes (edited) 
just those changes in that migration file
so, don't just dropTableI. Undo the changes in this migration file.

Comment (ogwurujohnson) [reply data/migrations/20190828030942_alter-organization-table.js]: resolved
